.pre-commit-config.yaml.jinja

```
@@ -20,6 +22,14 @@ repos:
         entry: found forbidden files; remove them
         language: fail
         files: "\\.rej$"
+  - repo: https://github.com/oca/maintainer-tools
+    rev: f1b5595 # Leave as is for stability
```

That comment is a bit stupid. If it were for that, we'd leave as it is all hooks :shrug: 

```suggestion
    rev: f1b5595
```

---

Well, what I meant was that, in this particular hook, we don't have any branches or tags to refer to, and would have to always pick from master. By referring to a particular commit, we make sure we don't get unexpected changes, thus "increasing stability".
In all the other hooks we don't need to because we refer to tagged versions.

I have removed it anyway :smile: